client/state/selectors/get-reader-followed-tags.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { filter, sortBy } from 'lodash';
+
+/**
+ * Returns all of the reader tags a user is following
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Array}          Reader Tags
+ */
+export default function getReaderFollowedTags( state ) {
+	return state.reader.tags.items
```

Comment:
that makes sense! its why I wrote https://github.com/Automattic/wp-calypso/pull/11733/commits/68743da8c082b3c3092aac3f6805b4b7811a7a1e which I can probably revert if i fix this.  

great catch! Thanks